### PR TITLE
(imp) [SD-3033] Dateline value should update with edits to the point of publishing

### PR DESCRIPTION
**Pull request by sivakuna-aap**
Requires [Superdesk-Core's PR#26](https://github.com/superdesk/superdesk-core/pull/26)

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
ping @superdesk/developers

File: client/app/scripts/superdesk-authoring/authoring.js [hunk 1]
```diff
@@ -126,7 +126,7 @@
          * Autosave an item
          */
         this.save = function saveAutosave(item) {
-            if (item._editable && item._locked) {
+            if (item._editable && !item._locked) {
```
Comment (ioanpocol) [inline client/app/scripts/superdesk-authoring/authoring.js]: Autosave works when an item is edited but when an item it will be locked.
So why is this change required?

Comment (sivakuna-aap) [inline client/app/scripts/superdesk-authoring/authoring.js]: That's not always true. There are some watches on `item` which sets field values using javascript. Once the values are set the `autosave.save()` is triggered even though the item isn't opened for edit. So this check prevents autosaving if the item isn't opened for edit and isn't locked by other users.

**Comment by akintolga** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:
